docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -242,6 +238,24 @@ When you submit a documentation pull request, automated tests are run. Those sam
 
 It is recommended to run tests on a clean copy of the repository, which is the purpose of the ``make clean`` command.
 
+Verifying your documentation PR in GitHub
+------------------------------------------
+
+Once you open your documentation PR, GitHub runs a series of automated checks on the PR. 
+One of those checks publishes the PR to a test site so you can visually see the results. 
+To view this test site for your PR:
+
+#. Scroll to the checks section of your PR on GitHub.
+#. Click :guilabel:`Show all checks`.
```

Comment:
I didn't know about the `guilabel` class. Cool!

Reply:
@gotmax23 nit: that's not a class but a role. There's a number of them listed @ https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-guilabel. It's not in doctutils, but is Sphinx-specific.